src/libsync/vfs/cfapi/cfapiwrapper.cpp

```diff
@@ -291,8 +297,147 @@ OCC::Optional<OCC::PinStateEnums::PinState> OCC::CfApiWrapper::PlaceHolderInfo::
     return cfPinStateToPinState(_data->PinState);
 }
 
-OCC::Result<void, QString> OCC::CfApiWrapper::registerSyncRoot(const QString &path, const QString &providerName, const QString &providerVersion)
+QString convertSidToStringSid(void *sid)
```

Comment:
This is used in retrieveWindowsSID only, so I'd either move it's content there or go the other way around completely and split a getTokenInformation out of retrieveWindowsSID. retriveWindowsSID would then become two simple calls.hink I think I prefer the latter actually. :-)

Reply:
@er-vin Ok. It's the latter then.